README.md

```diff
@@ -59,7 +59,7 @@ docker run  -d \
 			--volume ~/magic_mirror/config:/opt/magic_mirror/config \
 			--volume ~/magic_mirror/modules:/opt/magic_mirror/modules \
 			--name magic_mirror \
-			MichMich/MagicMirror
+			michmich/magicmirror
```

Comment:
Hi. I've tried to build a docker image based on this Dockerfile and it fails:
```
Sending build context to Docker daemon 7.095 MB
Step 1/12 : FROM izone/arm:node
node: Pulling from izone/arm
c5223e92d61a: Pull complete
1360befcec1b: Pull complete
Digest: sha256:2d7ef24f6bdf6c7e462587c2054755f383f8f35ea73bf3aba74f68859b7610b2
Status: Downloaded newer image for izone/arm:node
 ---> 4a6b73a00ecb
Step 2/12 : ENV NODE_ENV production
 ---> Running in f729f4b4891d
 ---> f2f5459d46a2
Removing intermediate container f729f4b4891d
Step 3/12 : ENV MM_PORT 8080
 ---> Running in 394f3d8d009f
 ---> 94571211dd56
Removing intermediate container 394f3d8d009f
Step 4/12 : WORKDIR /opt/magic_mirror
 ---> 54fba0cc5e8c
Removing intermediate container 50ff50c69f23
Step 5/12 : COPY package.json /opt/magic_mirror
 ---> 87954b5aa71b
Removing intermediate container 64a25a5d5b66
Step 6/12 : RUN npm install
 ---> Running in 009897841bcf
standard_init_linux.go:178: exec user process caused "exec format error"
The command '/bin/sh -c npm install' returned a non-zero code: 1
```
My setup is: Windows 7 64bit, docker Server Version: 17.04.0-ce

While current master's Dockerfile ( https://github.com/MichMich/MagicMirror/commit/25b5daf6a5860e9d2ce4c55a6b9d33143a9800e9 ) builds fine